openassessment/xblock/data_conversion.py

```diff
@@ -56,13 +57,42 @@ def convert_training_examples_list_to_dict(examples_list):
     ]
 
 
-def create_rubric_dict(prompt, criteria):
+def create_prompts_list(prompt_or_serialized_prompts):
+    """
+    Construct a list of prompts.
+
+    Initially a block had a single prompt which was saved as a simple string.
+    In that case a new prompt dict is constructed from it.
+
+    Args:
+        prompt_or_serialized_prompts (unicode): A string which can either
+        be a single prompt text or json for a list of prompts.
+
+    Returns:
+        list of dict
+    """
+
+    if prompt_or_serialized_prompts is None:
+        prompt_or_serialized_prompts = ''
+
+    try:
+        prompts = json.loads(prompt_or_serialized_prompts)
```

Comment:
Is this to say, if we get a single prompt instead of a list of prompts, json.loads will throw an exception, and we'll then assume it is one prompt?

I think I'd prefer to determine if we have one prompt, or a set of serialized prompts, before we hit this line, and if there isa  ValueError, it would only be a legitimate one. 

If there is a single prompt, the serialized prompt list given to this function would have the one element.